Makefile

```diff
@@ -193,7 +189,7 @@ clean-generated:
 	-rm -rf ./client
 	-rm -rf ./swagger/
 	-rm -f ./migration/sqlbindata.go
-	-rm -f ./template/bindata.go
+	-rm -f ./environment/generated/templates.go
```

Comment:
deleting file would be enough or deleting folder `generated` would be better.

Reply:
I kept it consistent - the same thing is done for the other files.  the environment/generated/ dir is in .gitignore anyway